pom.xml

```diff
@@ -28,7 +28,7 @@
     <groupId>org.xwiki.commons</groupId>
     <artifactId>xwiki-commons-pom</artifactId>
     <!-- The application meeting use the new Mail Sender API which bundled by default in XWiki Enterprise since 6.2M1 -->
-    <version>6.2-milestone-1</version>
+    <version>6.4-milestone-3</version>
```

Comment:
We shouldn't depend on such a recent version and since the new Mail Sender API is going to be merged in the 6.2.x branch for 6.2.5 we should depend on 6.2-SNAPSHOT till 6.2.5 is released. Since the merge has not been done yet, it's ok to temporarily depend on 6.4M3 but this needs to be modified absolutely. Thus this requires a big comment before it can be committed!